### PR TITLE
Minor release 1.11

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` is updated
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Documentation in `docs` is updated

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 import sys
 
-version = "1.10.2"
+version = "1.11dev"
```
Comment (ewels) [inline setup.py:6]: Need to bump the version number here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # nf-core/tools: Changelog
 
+## v1.11dev
```
Comment (ewels) [inline CHANGELOG.md:3]: Need to bump the version number here

**Comment by ewels** [comment]
Will probably need to prep for a manual fix to https://github.com/nf-core/tools/issues/410 for the release too 🤔

**Comment by ggabernet** [comment]
maybe we wait till tomorrow for the release but just marked it as ready for review in case someone wants to check it already today ;)

**Comment by ewels** [comment]
I'm happy with pushing the release tomorrow morning 🚀

**Review by ewels — APPROVED** [review]
Looking good I think!

File: nf_core/lint.py [hunk 3]
```diff
@@ -790,32 +815,27 @@ def check_actions_awstest(self):
             with open(fn, "r") as fh:
                 wf = yaml.safe_load(fh)
 
-            # Check that the action is only turned on for push
+            # Check that the action is only turned on for workflow_dispatch
             try:
-                assert "push" in wf[True]
+                assert "push" not in wf[True]
                 assert "pull_request" not in wf[True]
+                assert "workflow_dispatch" in wf[True]
```
Comment (ggabernet) [inline nf_core/lint.py:822]:
```suggestion
                assert "workflow_dispatch" in wf[True]
                assert "push" not in wf[True]
                assert "pull_request" not in wf[True]
```

Comment (ggabernet) [reply nf_core/lint.py:822]: hmm I thought that this was already re-ordered in my PR

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
 # nf-core/tools: Changelog
 
+## v1.11.0
```
Comment (apeltzer) [inline CHANGELOG.md:3]: Just codename required

**Review by apeltzer — APPROVED** [review]
Looking great - thank you 🙇

File: CHANGELOG.md (hunk 4 above)
Comment (ewels) [reply CHANGELOG.md:3]: https://github.com/nf-core/tools/pull/756

File: nf_core/lint.py (hunk 3 above)
Comment (ewels) [reply nf_core/lint.py:822]: Yes, I thought so too? 🤔

Comment (ewels) [reply nf_core/lint.py:822]: Must have been a different place I think - here it's just one `in`, in the discussion before I remember that there were 2.

**Comment by ggabernet** [comment]
What about the nf-core base docker container for `v1.11` then? I am not sure where this is created and wanted to make sure it says `1.11` and not `1.11.0`

**Pull request merged**